lib/rules/keyframe-selector-notation/README.md

```diff
@@ -0,0 +1,54 @@
+# keyframe-selector-notation
+
+Specify keyword or percentage notation for keyframe selectors. This rule only enforces the convention for `from`, `to`, `0%` and `100%`.
```

Comment:
```suggestion
Specify keyword or percentage notation for keyframe selectors.
```

Let's be consistent with other descriptions (and in sync with the list of rules). 

FYI, "This rule..." stuff usually sits in the [Expanded description](https://stylelint.io/user-guide/rules/about#readmes). No need this move this particular one down though as it's redundant.